UPtables/db/seeds.rb

```diff
@@ -104,6 +104,10 @@
 EctsModule.create(:name => "Theoretische Grundlagen: Effiziente Algorithmen")
 EctsModule.create(:name => "Konzepte paralleler Programmierung")
 
+Curriculum.create(:name => "Informatik Bachelor")
+Curriculum.create(:name => "Informatik Master")
+Curriculum.create(:name => "CS Bachelor")
+Curriculum.create(:name => "CS Master")
```

Comment:
@felixKubicek could you seed some associated curricula please? :kissing_closed_eyes: